### PR TITLE
Translation of ecommerce chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Part of #586 

Please review!!

### CODE REVIEW

File: src/content/ja/2019/ecommerce.md [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+
+## タイプ別のリクエストとペイロード
+
+以下の表は、デスクトップでのリクエストの場合のものです。
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Type</th>
```
Comment (bazzadp) [inline src/content/ja/2019/ecommerce.md:190]: Should we translate type (aka filetype)?
```suggestion
        <th>ファイルの種類</th>
```

File: src/content/ja/2019/ecommerce.md [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+      <tr>
+        <td>フォント</td>
+        <td class="numeric">47</td>
+        <td class="numeric">85</td>
+        <td class="numeric">144</td>
+        <td class="numeric">226</td>
+        <td class="numeric">339</td>
+      </tr>
+      <tr>
+        <td>css</td>
```
Comment (bazzadp) [inline src/content/ja/2019/ecommerce.md:232]: Minor nit, but it seems a little off to have others translated but not this or CSS. Either we should say they all are technical terms (filetypes) and not translate, or they are phrases and so should translate?

What do you think?

"Other" though is definitely not a technical term so should be translated in my opinion.

File: src/content/ja/2019/ecommerce.md [hunk 3]
```diff
@@ -0,0 +1,608 @@
...
+    </tbody>
+  </table>
+<figcaption>図9. リソースタイプ別のページ重量分布（KB単位）のパーセンタイル。</figcaption>
+</figure>
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Type</th>
```
Comment (bazzadp) [inline src/content/ja/2019/ecommerce.md:288]:
```suggestion
        <th>ファイルの種類</th>
```

File: src/content/ja/2019/ecommerce.md [hunk 4]
```diff
@@ -0,0 +1,608 @@
...
+        <td class="numeric">3</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>図10リソースタイプ別の1ページあたりのリクエストの分布のパーセンタイル。</figcaption>
+</figure>
+
+Eコマースページでは、画像が最大のリクエスト数とバイト数の割合を占めています。デスクトップEコマースページの中央値には、1,514KB(1.5MB)の重さの画像が39枚含まれています。
+
+[JavaScript](./javascript)リクエストの数は、より良いバンドル(および/または[HTTP/2](./http2)多重化)によってパフォーマンスを向上する可能性があることを示しています。JavaScriptファイルの総バイト数はそれほど大きくありませんが、個別のリクエストが多くなっています。[HTTP/2](./http2#user-content-adoption-of-http2)の章によると、リクエストの40％以上はHTTP/2経由ではないそうです。同様に、CSSファイルは3番目にリクエスト数が多いですが、一般的には少ないです。CSSファイル(またはHTTP/2)をマージすることで、そのようなサイトのパフォーマンスを向上させることができるかもしれません。著者の経験では、多くのEコマースページでは、未使用のCSSとJavaScriptの割合が高い。[Videos](./media) のリクエスト数は少ないかもしれませんが、(驚くことではありません) 特にペイロードが重いサイトでは、ページの重量の割合が高くなります。
```
Comment (bazzadp) [inline src/content/ja/2019/ecommerce.md:384]: Should we translate "Videos"?

File: src/content/ja/2019/ecommerce.md [hunk 5]
```diff
@@ -0,0 +1,608 @@
...
+<p class="note">私たちのデータ収集<a href="./methodology">方法論</a>はクリックやスクロールなど、ページ上でのユーザー操作をシミュレートしていないため、遅延して読み込まれた画像はこれらの結果には表示されないことに注意してください。</p>
+
+上記の図12と13で中央値のEコマースページには、モバイルでは37枚の画像と1,517KBの画像ペイロードがあり、デスクトップでは40枚の画像と1,524KBの画像ペイロードがあることを示しています。ホームページの10％は、90以上の画像と6MB近くの画像ペイロードを持っています！
+
+<figure>
+  <div class="big-number">1,517 KB</div>
+  <figcaption>図14. モバイルEコマースページあたりの画像バイト数の中央値。</figcaption>
+</figure>
+
+Eコマースページのかなりの割合で、大きな画像ペイロードを持ち、最初のロード時に大量の画像リクエストを行います。詳細については、HTTP Archiveの[State of Images](https://httparchive.org/reports/state-of-images)レポート、および[media](./media)と[page weight](./page weight)の章を参照してください。
```
Comment (bazzadp) [inline src/content/ja/2019/ecommerce.md:427]: Are we happy leaving the english terms untranslated in this sentence?

File: src/content/ja/2019/ecommerce.md [hunk 6]
```diff
@@ -0,0 +1,608 @@
+---
+part_number: III
+chapter_number: 13
+title: Ecommerce
```
Comment (bazzadp) [inline src/content/ja/2019/ecommerce.md:4]:
```suggestion
title: Eコマース
```

File: src/content/ja/2019/ecommerce.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2019/ecommerce.md:190]: Thanks, I fixed.

File: src/content/ja/2019/ecommerce.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2019/ecommerce.md:232]: I think that don't translate css, html.
Maybe Allmost japanese think so.

File: src/content/ja/2019/ecommerce.md (hunk 4 above)
Comment (ksakae1216) [reply src/content/ja/2019/ecommerce.md:384]: Thanks, I mistake, so fixed.

File: src/content/ja/2019/ecommerce.md (hunk 3 above)
Comment (ksakae1216) [reply src/content/ja/2019/ecommerce.md]: I fixed.

File: src/content/ja/2019/ecommerce.md (hunk 6 above)
Comment (ksakae1216) [reply src/content/ja/2019/ecommerce.md]: I fixed.

**Comment by ksakae1216** [comment]
@bazzadp hi.

I translated alankent and samdutton's PR.

I have a question.
Do add my PR too?

**Comment by bazzadp** [comment]
> @bazzadp hi.
> 
> I translated alankent and samdutton's PR.

Thanks! Add to this pull request and I will have a look.

> I have a question.
> Do add my PR too?

Do you mean your bio? Sorry only Aithors get bios - see #350

**Comment by ksakae1216** [comment]
@bazzadp Thanks, I see, I merged this pull-req .

**Comment by bazzadp** [comment]
I don't see the author bios in this branch even though you said you'd translated them?

**Comment by ksakae1216** [comment]
@bazzadp , sorry.
translate PR.

File: src/templates/ja/2019/base.html [hunk 7]
```diff
@@ -102,7 +102,7 @@
...
   "argyleink": "<i lang=\"en\">Adam Argyle is a Google Chrome developer relations member focused on CSS; He's a web addict with an insatiable lust for great UX &amp; UI; Find him on the web <a href=\"https://twitter.com/argyleink\">@argyleink</a> or checkout his website <a href=\"https://nerdy.dev\">https://nerdy.dev</a>;</i>",
-  "alankent": "<i lang=\"en\">Alan Kent is a Developer Advocate at Google focusing on e-commerce and content ecosystems. He blogs at alankent.me and tweets as @akent99.</i>",
+  "alankent": "<i lang=\"ja\">Alan KentはGoogleのDeveloper Advocateで、Eコマースとコンテンツエコシステムに焦点を当てています。彼は alankent.me でブログを書いており、@akent99 としてツイートしています。</i>",
```
Comment (bazzadp) [inline src/templates/ja/2019/base.html:105]: You can remove the lang parts of this. Only needed when not in the lang of the rest of the document:

```suggestion
  "alankent": "Alan KentはGoogleのDeveloper Advocateで、Eコマースとコンテンツエコシステムに焦点を当てています。彼は alankent.me でブログを書いており、@akent99 としてツイートしています",
```

File: src/templates/ja/2019/base.html [hunk 8]
```diff
@@ -121,7 +121,7 @@
...
-  "samdutton": "<i lang=\"en\">Sam Dutton has worked with the Google Chrome team as a Developer Advocate since 2011. He has organized and presented at a number of events, created and taught several web development courses, and worked on a range of videos, codelabs and written guidance covering PWA, performance, media, image and 'Next Billion Users' initiatives. He maintains <a href=\"https://simpl.info\">simpl.info</a>, which provides simplest possible examples of HTML, CSS and JavaScript. Sam grew up in South Australia, went to university in Sydney, and has lived since 1986 in London.</i>",
+  "samdutton": "<i lang=\"ja\">Sam Duttonは2011年からDeveloper AdvocateとしてGoogle Chromeチームで働いています。数々のイベントを企画して発表し、いくつかのウェブ開発コースを作成して教え、PWA、パフォーマンス、メディア、イメージ、「Next Billion Users」イニシアティブをカバーする様々なビデオ、コードラボ、文書化されたガイダンスに取り組んできました。彼は<a href="htt ps://simpl.info\">simpl.info</a>を管理しており、HTML、CSS、JavaScriptの最もシンプルな例を提供しています。南オーストラリアで育ち、シドニーの大学に進学し、1986年からロンドンに住んでいます。</i>",
```
Comment (bazzadp) [inline src/templates/ja/2019/base.html:124]: Same here:
```suggestion
  "samdutton": "Sam Duttonは2011年からDeveloper AdvocateとしてGoogle Chromeチームで働いています。数々のイベントを企画して発表し、いくつかのウェブ開発コースを作成して教え、PWA、パフォーマンス、メディア、イメージ、「Next Billion Users」イニシアティブをカバーする様々なビデオ、コードラボ、文書化されたガイダンスに取り組んできました。彼は<a href="htt ps://simpl.info\">simpl.info</a>を管理しており、HTML、CSS、JavaScriptの最もシンプルな例を提供しています。南オーストラリアで育ち、シドニーの大学に進学し、1986年からロンドンに住んでいます。",
```

File: src/templates/ja/2019/base.html (hunk 7 above)
Comment (ksakae1216) [reply src/templates/ja/2019/base.html:105]: delete tag.

File: src/templates/ja/2019/base.html (hunk 8 above)
Comment (ksakae1216) [reply src/templates/ja/2019/base.html:124]: delete tag.

**Comment by ksakae1216** [comment]
@bazzadp , I have a question.

Should I also translate featured_chapters.html for the css, cdn, etc that I've been translating?
If so, create a other branch and translate it.

**Comment by bazzadp** [comment]
Brilliant! Thanks so much @ksakae1216 for all these extra changes. Think it would be good to do these going forward. Will save effort to revisit later and makes sense to do at same time as chapter in my opinion.

> Should I also translate featured_chapters.html for the css, cdn, etc that I've been translating?
If so, create a other branch and translate it.

That would be great! And also translating the relevant Bios? Was planning on asking this to be done at end after the chapters, but think it would be better if we did it as we went along? Means the chapter looks more completely translated.

**Comment by ksakae1216** [comment]
@bazzadp 

OK, I create other branch and translate featured_chapters.html for the css, cdn, etc.
In the future, I translate chapter when translate featured_chapters.html too.

**Pull request merged**